### DISCUSSION TITLE
Tutorial Paraview visualisation error due to duplicate names

### DISCUSSION THREAD

**Discussion by tannhorn** [neams-th-coe/cardinal#731] [Q&A] [answered]
Hello,

When going through the sfr_7pin tutorial, I cannot view the results (solid_out.e) in paraview with the following error message:

ERROR: Duplicate names detected.
       The name 'duct_outer' was found for both SideSet 10000 and NodeSet 10000.
       Names must be unique over all types in a finite element model.

**Comment by tannhorn** [comment]
Appears that this happens only when I open the file with paraview from WSL, so I guess not much you can do about it. Using the paraview from within Windows works.

**Reply by aprilnovak** [reply, marked as answer]
Hi @Tannhorn, yes this is a known issue with Paraview release 5.10.0 (many MOOSE users have encountered it, https://discourse.paraview.org/t/unique-name-requirement-for-nodesets-and-sidesets/8171/2). 

One way around it is to load the "Legacy" plugin in Paraview. If you go to the "Tools" drop-down menu, select "Manage Plugins" and then load the "LegacyExodusReader" plugin. 

I would guess that you probably have different paraview versions in/outside of WSL.

**Reply by tannhorn** [reply]
I see, thanks. Good news is that my Windows paraview is 5.11.1 so that this issue might be actually getting fixed with the next release.
